### PR TITLE
UI fixes

### PR DESCRIPTION
**Pull request by alex-melnik**
- adjust theme inside widgets;
- remove spacings from dashboard, add background for widgets;
- minor refactoring for performance optimization

### CODE REVIEW

File: packages/bootstrap/stories/widgets/TopContributorsContext.tsx [hunk 1]
```diff
@@ -6,6 +6,8 @@ import {
   smeConfig,
   createClient,
 } from '../../src/components/mockData';
+import { ThemeProvider } from 'styled-components';
```
Comment (i-kushlianski) [inline packages/bootstrap/stories/widgets/TopContributorsContext.tsx]: import

Comment (alex-melnik) [reply packages/bootstrap/stories/widgets/TopContributorsContext.tsx]: Done

File: packages/bootstrap/src/components/dashboard/Dashboard.tsx [hunk 2]
```diff
@@ -32,25 +32,28 @@ export interface DashboardProps {
 export const Dashboard: React.FC<DashboardProps> = (props) => {
   const { client, provider, config } = props;
 
+  const theme = useMemo(
+    () => createTheme({ ...initialDashboardTheme, ...config.theme }),
+    [config.theme],
+  );
```
Comment (miraage) [inline packages/bootstrap/src/components/dashboard/Dashboard.tsx]: The theme object itself should be created outside components. Ideally, in Desktop. Is it possible to remove it from the Dashboard component?

Comment (alex-melnik) [reply packages/bootstrap/src/components/dashboard/Dashboard.tsx]: Right now theme is coming from desktop (in `.json` file with set of widgets), and configuring (colors and its grayscales) in hiro-ui.  We can pass colors, sizes, etc (or don't pass at all), and theme will be configured in dashboard.

**Pull request merged**